pkg/client/dtclient/entities.go

```diff
@@ -40,6 +40,16 @@ type KeyUserActionsWebResponse struct {
 	} `json:"keyUserActionList"`
 }
 
+type UserActionAndSessionPropertyEntry struct {
```

Comment:
I don't see the benefit of declaring these structures in a separate file when they are used only once. I suggest putting them closer to the usage, even using them as an unnamed type.